### ISSUE TITLE
Methodology - women's retirement age - Labour force survey

### ISSUE THREAD

**Issue by grace-spitzer-wong** [GSS-Cogs/family-disability#48]
To add methodology notes to Labour force survey DDA data  - women's retirement age was sooner. Methodology info to add as data isn't comparable - survey questions different and change in legislation from Disability Discrimination Act to Equality Act.

Linked to https://github.com/GSS-Cogs/family-disability/issues/3

**Comment by ajtucker** [comment]
This is potentially a question for UR (@JohnLewisUR) too. In the metadata for a dataset -- see for example http://gss-data.org.uk/dataset?uri=http%3A%2F%2Fgss-data.org.uk%2Fdata%2Fgss_data%2Fdisability%2Fons-labour-force-survey -- we could/should provide a link to the methodology document(s).

**Comment by JohnLewisUR** [comment]
Yes, users want a link to methodology documentation with the data. They also want a host of other info too, to help them understand how to use the data, and what it can and can't be used for, etc - but we'd have to get that directly from producers, as this kind of info doesn't really exist at present.

**Comment by ajtucker** [comment]
This is a general issue for all datasets. I've added an issue to the backlog for GSS-Cogs/gss-utils#21, as that's where we're managing the metadata, but we'll also need some support in PMD for how the metadata is shown to the user.
